Flow.Launcher/PublicAPIInstance.cs

```diff
@@ -103,6 +103,16 @@ public void OpenSettingDialog()
             });
         }
 
+        public void ShellRun(string cmd)
+        {
+            System.Diagnostics.Process process = new();
+            var startInfo = process.StartInfo;
+            startInfo.FileName = "cmd.exe";
+            startInfo.Arguments = $"/C {cmd}";
+            startInfo.CreateNoWindow = true;
+            process.Start();
```

Comment:
@spedrickson have you tested this with your plugin? the method is not returning the result from the command

Reply:
Yes it works for my plugin, but I'm only using it to launch files (identical to Browser plugin), not receive data back on the plugin side. 

One concern with returning results: right now the commands are launched in a separate process to prevent locking up the Flow UI, so even long-running commands won't impact user experience. Maintaining a list of running processes is easy enough, but then we get into design decisions like whether return order should be maintained. 

Might be worth adding a separate function like `GetShellRun` ,that works the same way but keeps the process and regularly polls it for output?